modules/worker-utils/src/lib/worker-farm/worker-thread.ts

```diff
@@ -28,7 +28,10 @@ export default class WorkerThread {
 
   /** Checks if workers are supported on this platform */
   static isSupported(): boolean {
-    return (typeof Worker !== 'undefined' && isBrowser) || typeof NodeWorker !== undefined;
+    return (
+      (typeof Worker !== 'undefined' && isBrowser) ||
+      (typeof NodeWorker !== undefined && !isBrowser)
```

Comment:
Is NodeWorker defined in the browser?

Reply:
Yes it is always function. 
![image](https://user-images.githubusercontent.com/70207219/185356855-07aec60b-9abf-45d0-a43e-ba95cbdaea78.png)